app/components/Send/SendPanel/SendRecipientList/SendRecipientListItem/index.jsx

```diff
@@ -48,6 +59,26 @@ class SendRecipientListItem extends Component<Props> {
 
     const { name } = e.target
     let { value } = e.target
+    if (name === 'address') this.setState({ nnsError: '' })
```

Comment:
I think its easier to read if we use brackets around all if statements, but I don't want to enforce my personal preferences. Perhaps we should come up with some sort of code style guide for neon-wallet that everyone agrees upon. @comountainclimber what do you think?

Reply:
We actually enforce very specific linting and formatting rules **already** via eslint and prettier... While I agree if else blocks without the `{` are slightly weird - they are perfectly valid within the context of our style guide (although I would be open to excluding them)

Reply:
It is also my personal preference to always have brackets around an if statement. However, I think in the context of this open source repo, it's more important to keep consistent with the code styles already present in the repo.

I originally had the following as my personal preference, but the linter does not allow it:
`name === 'address' && this.setState({ nnsError: '' })`